packages/release/release-repart-local.conf

```diff
@@ -1,4 +1,6 @@
 [Partition]
+Label=BOTTLEROCKET-DATA
+
 # This is the partition type UUID for BOTTLEROCKET-DATA, which will be resized
 # to fill the remaining sectors on the disk where it resides.
 Type=626f7474-6c65-6474-6861-726d61726b73
```

Comment:
I decided to use the same partition type UUID for both the fallback data partition and the preferred data partition since I wasn't not sure if there's a benefit to having it be different.

Reply:
I can also add `SizeMinBytes` to ensure a reasonable sized data partition is created or fail otherwise.

Reply:
That seems like a good safety protection.

Reply:
There is a default minimum size constraint of 10 MB imposed already. Maybe that's sufficient? Who's to say a user can't have a 10 MB data partition for all their needs. Maybe they want to rely on ephemeral storage devices/local instance storage.